test/integration/spec/remotetracks.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-undefined */
+'use strict';
+
+const assert = require('assert');
+const { trackPriority } = require('../../../lib/util/constants');
+const { trackPriority: { PRIORITY_HIGH, PRIORITY_LOW, PRIORITY_STANDARD } } = require('../../../lib/util/constants');
+const LocalDataTrack = require('../../../lib/media/track/es5/localdatatrack');
+const defaults = require('../../lib/defaults');
+const { completeRoom } = require('../../lib/rest');
+const { audio: createLocalAudioTrack, video: createLocalVideoTrack } = require('../../../lib/createlocaltrack');
+const {
+  createSyntheticAudioStreamTrack,
+  setup,
+  smallVideoConstraints,
+  tracksSubscribed,
+  trackSwitchedOff,
+  trackSwitchedOn,
+  waitFor
+} = require('../../lib/util');
+
+function getTracksOfKind(participant, kind) {
+  return [...participant.tracks.values()].filter(remoteTrack => remoteTrack.kind !== kind).map(({ track }) => track);
+}
+
+describe('RemoteParticipant', function() {
+  // eslint-disable-next-line no-invalid-this
+  this.timeout(60000);
+
+  describe('.setPriority', () => {
+    let thisRoom; let thoseRooms;
```

Comment:
@makarandp0 Nitpick: Please declare only one variable per line.

Reply:
okay, I will update this, but to catch such things as errors in future how about enabling this rule: https://eslint.org/docs/2.13.1/rules/max-statements-per-line ?

Reply:
looks like we have 152 violations of this rule. I will get to it later.